lib/resources.rb

```diff
@@ -1453,27 +1453,163 @@ def get_resource(key)
-                                  "encounter"=> {"reference"=> "Encounter/2571909"},
-                                  "conclusion"=> "Report Here is a rad   Signature Line:***** Preliminary Report *****   Transcribed by: SM"}}]
+        "resourceType" => "Bundle",
+        "id" => "c7d2abdd-0c20-433e-a9a9-53e655f79885",
+        "type" => "searchset",
+        "base" => "https://fhir-open.sandboxcernerpowerchart.com/may2015/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
+        "total" => 3,
+        "link" => [
+            {
+                "relation" => "self",
+                "url" => "https://fhir-open.sandboxcernerpowerchart.com/may2015/d075cf8b-3261-481d-97e5-ba6c48d3b41f/DiagnosticReport?patient=1316020"
+            }
+        ],
+        "entry" => [
+            {
+                "resource" => {
+                    "resourceType" => "DiagnosticReport",
+                    "id" => "5153487",
+                    "meta" => {
+                        "versionId" => "3"
+                    },
+                    "text" => {
+                        "status" => "generated",
+                        "div" => "&lt;div>&lt;p>&lt;b>Diagnostic Report&lt;/b>&lt;/p>&lt;p>&lt;b>Service Category&lt;/b>: Radiology&lt;/p>&lt;p>&lt;b>Name&lt;/b>: XR Wrist Complete Left&lt;/p>&lt;p>&lt;b>Performer&lt;/b>: Who, Doctor&lt;/p>&lt;p>&lt;b>Status&lt;/b>: Final&lt;/p>&lt;/div>"
+                    },
+                    "name" => {
+                        "coding" => [
+                            {
+                                "system" => "http://snomed.info/sct",
+                                "code" => "169074008"
+                            }
+                        ],
+                        "text" => "XR Wrist Complete Left"
+                    },
+                    "status" => "final",
+                    "issued" => "2015-08-06T20:03:21.000Z",
+                    "subject" => {
+                        "reference" => "Patient/1316020"
+                    },
+                    "performer" => {
+                        "reference" => "Practitioner/2048007",
+                        "display" => "Who, Doctor"
+                    },
+                    "encounter" => {
+                        "reference" => "Encounter/2787906"
+                    },
+                    "serviceCategory" => {
+                        "coding" => [
+                            {
+                                "system" => "http://hl7.org/fhir/v2/0074",
+                                "code" => "RAD",
+                                "display" => "Radiology"
+                            }
+                        ],
+                        "text" => "Radiology"
+                    },
+                    "diagnosticDateTime" => "2015-08-06T20:03:09.000Z",
+                    "conclusion" => "Report_asdfasdfasdf jjimmmy   Signature Line:***** Preliminary Report *****   Transcribed by: DW      Reason for exam:4061979"
```

Comment:
Since this is going to be publicly visible, should we clean up some of these values?